src/containers/guest-home-page/signup-dialog/SignupDialog.jsx

```diff
@@ -111,7 +119,7 @@ const SignupDialog = ({ type }) => {
             handleSubmit={handleSubmit}
           />
           <GoogleLogin
-            buttonWidth={styles.form.maxWidth}
+            buttonWidth={getButtonWidth()}
```

Comment:
If you agree with object style: 
```suggestion
            buttonWidth={getButtonWidth[sizeName]}
```